### PR TITLE
Add new documents for Tizen Image Creator 

### PR DESCRIPTION
**Pull request by jay4peace**
### Change Description ###

- Add new documents for Tizen Image Creator (TIC)
- TIC is new tool to create Tizen binary image

### CODE REVIEW

**Review by sagarkedari — CHANGES_REQUESTED** [review]
@jay4peace, I have reviewed the files and added my feedback comments. Kindly check and incorporate the changes.

File: docs/platform/developing/creating-tic.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+- provides web-based user interface
+- easy installation with the docker image
+- YAML style recipe is supported
+
+### Supported Environment
+
+- Ubuntu 16.04 LTS or Later
+
+## How to use TIC
```
[the remark this answers is not in the document]
Comment (jay4peace) [reply docs/platform/developing/creating-tic.md:21]: This section is for setting up TIC on user's PC, so I think, `Set up TIC on your PC` is better than `Create a Tizen image using TIC`

File: docs/platform/developing/creating-tic.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+   ```shell
+   docker login -u [DockerHub_Username] -p [DockerHub_Password]
+   docker-compose pull
+   docker-compose up -d
+   ```
+
+## Connect to TIC web-service
+
+- Open any browser with <http://127.0.0.1:8088>
```
[the remark this answers is not in the document]
Comment (jay4peace) [reply docs/platform/developing/creating-tic.md:43]: `http://127.0.0.1:8088` is the address for the TIC service on local host PC

**Review by sagarkedari — CHANGES_REQUESTED** [review]
@jay4peace , I have reviewed and provided my additional feedback comments. Kindly check and incorporate the changes.

**Review by sagarkedari — CHANGES_REQUESTED** [review]
@jay4peace, Please incorporate the requested changes.

**Review by sagarkedari — APPROVED** [review]
@annie-samsung , This PR is approved from my end.

**Comment by jay4peace** [comment]
@annie-samsung , @journey2w Is there anything I have to do to proceed?

**Comment by jay4peace** [comment]
@mijin85cho @annie-samsung @journey2w Please move on to next step

**Comment by mijin85cho** [comment]
@annie-samsung Would you please check this PR? If there is no more issue, how about merging it?

**Pull request merged**